moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -554,29 +567,6 @@ TEST(FK, OneRobot)
   EXPECT_NEAR_TRACED(state.getGlobalLinkTransform("link_e").translation(), Eigen::Vector3d(2.8, 0.6, 0));
 }
 
-class LoadPR2 : public testing::Test
```

Comment:
The PR2-based test is replaced by one exploiting `MODEL2`.

Reply:
nice!